app/src/main/java/at/irfc/app/ui/program/ProgramScreen.kt

```diff
@@ -55,14 +63,82 @@ fun ProgramScreen(
 
             eventListResource.data?.let { eventList ->
                 items(eventList, EventWithDetails::id) { event ->
-                    Text(
-                        text = event.title,
+                    Card(
+                        shape = RoundedCornerShape(20.dp),
+                        elevation = CardDefaults.cardElevation(
+                            defaultElevation = 10.dp
+                        ),
                         modifier = Modifier
-                            .padding(5.dp)
-                            .clickable {
-                                navController.navigate(ProgramDetailScreenDestination(event.id))
+                            .padding(bottom = 16.dp)
+                            .fillMaxWidth()
+                    ) {
+                        Row(
+                            modifier = Modifier
+                                .clickable {
+                                    navController.navigate(
+                                        ProgramDetailScreenDestination(event.id)
+                                    )
+                                }
+                                .fillMaxWidth()
+                        ) {
+// Box for the image
+                            Box(
```

Comment:
There is probably no need for this Box, all the modifiers can also be applied to the AsyncImage.

For the modifiers please have a look at the other images (e.g. ProgramDetailScreen) to have the right behavior for sizing (crop,...)